R/zarr-array.R

```diff
@@ -16,71 +16,70 @@ ZarrArray <- R6::R6Class("ZarrArray",
     # store Array store, already initialized.
     #' @keywords internal
     store = NULL,
-    #' chunk_store Separate storage for chunks. If not provided, `store` will be used for storage of both chunks and metadata.
+    # chunk_store Separate storage for chunks. If not provided, `store` will be used for storage of both chunks and metadata.
```

Comment:
Why are the `#'` to `#` changes needed? I am not an expert in Roxygen comments but I see the apostrophe format at https://roxygen2.r-lib.org/articles/rd-other.html#r6

Reply:
Yeah... so with the `#'` on private methods, it does strange stuff.

![image](https://github.com/user-attachments/assets/90a45066-b224-478c-b6df-cdaf6e4513d0)